src/state/opamStateTypes.mli

```diff
@@ -119,6 +120,9 @@ type +'lock switch_state = {
   packages: package_set;
   (** The set of all known packages *)
 
+  sys_packages: sys_pkg_status package_map;
+  (** Map of package and their system dependencies packages status *)
```

Comment:
We'll need to make this lazy at some point

Reply:
lazied!